### PR TITLE
[Task 1855293] synchronize status of sql server among different views (explorer/properties view)

### PR DESCRIPTION
**Pull request by shenqianjin**
[AB#1855293](https://dev.azure.com/mseng/a4d27ce2-a42d-4b71-8eef-78cee9a9728e/_workitems/edit/1855293)

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/azure/toolkit/intellij/mysql/MySQLPropertyView.java [hunk 1]
```diff
@@ -106,6 +106,7 @@
         AzureEventBus.after("mysql|server.start", this::onMySqlServerStatusChanged);
         AzureEventBus.after("mysql|server.restart", this::onMySqlServerStatusChanged);
         AzureEventBus.after("mysql|server.stop", this::onMySqlServerStatusChanged);
+        AzureEventBus.after("mysql|server.delete", this::onMySqlServerStatusDeleted);
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/azure/toolkit/intellij/mysql/MySQLPropertyView.java:103]: image

Comment (shenqianjin) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/azure/toolkit/intellij/mysql/MySQLPropertyView.java:103]: I think it's not a good idea. `onMySqlServerStatusDeleted` keeps the same style with `onMySqlServerStatusDeleting`, just like `onMySqlServerStatusChanged` with `onMySqlServerStatusChanging`.

Comment (andxu) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/azure/toolkit/intellij/mysql/MySQLPropertyView.java:103]: already handled this case

Comment (shenqianjin) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/azure/toolkit/intellij/mysql/MySQLPropertyView.java:103]: please note the handle logic of `delete` is different with other actions. it close the property view after `delete` action finishes.

Comment (andxu) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/azure/toolkit/intellij/mysql/MySQLPropertyView.java:103]: even so, you need to delete line 100 to 105

Comment (shenqianjin) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/azure/toolkit/intellij/mysql/MySQLPropertyView.java:103]: deleted.

**Review by andxu — APPROVED** [review]
LGTM

**Pull request merged**